.changeset/gold-crabs-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-development-kitchen': minor
```

Comment:
since this changes the default spacing, it should be a major (any other current users may need to accommodate this change in their code, so it's not necessarily a drop-in for them)
```suggestion
'@guardian/source-development-kitchen': major
```